app/Http/Controllers/Frontend/DiscordController.php

```diff
@@ -32,11 +32,16 @@ public function fetch()
 
         if($this->getDiscordServerId()) {
             $discordFetch = Cache::remember('discordFetch', $seconds, function () {
-                $raw = file_get_contents(
-                    'https://discordapp.com/api/servers/' .
+                $raw = @file_get_contents(
```

Comment:
Putting an `@` in front is only for silencing any php error message. Better use try {} catch to show the User when something isn't working.